src/bin/list_store.rs

```diff
@@ -0,0 +1,299 @@
+    // Column for fixed toggles
+    {
+        let renderer = CellRendererToggle::new();
+        renderer.connect_toggled(move |w, path| fixed_toggled(&*model, w, path));
+        let column = TreeViewColumn::new();
+        column.pack_start(&renderer, true);
+        column.set_title("Fixed?");
+        column.add_attribute(&renderer, "active", COLUMN_FIXED);
+        column.set_sizing(TreeViewColumnSizing::Fixed);
+        column.set_fixed_width(50);
+        treeview.append_column(&column);
+    }
+
+    // Column for bug numbers
+    {
+        let renderer = CellRendererText::new();
+        let column = TreeViewColumn::new();
+        column.pack_start(&renderer, true);
+        column.set_title("Bug number");
+        column.add_attribute(&renderer, "text", COLUMN_NUMBER);
+        column.set_sort_column_id(COLUMN_NUMBER);
+        treeview.append_column(&column);
+    }
+
+    // Column for severities
+    {
+        let renderer = CellRendererText::new();
+        let column = TreeViewColumn::new();
+        column.pack_start(&renderer, true);
+        column.set_title("Severity");
+        column.add_attribute(&renderer, "text", COLUMN_SEVERITY);
+        column.set_sort_column_id(COLUMN_SEVERITY);
+        treeview.append_column(&column);
+    }
+
+    // Column for description
+    {
+        let renderer = CellRendererText::new();
+        let column = TreeViewColumn::new();
+        column.pack_start(&renderer, true);
+        column.set_title("Description");
+        column.add_attribute(&renderer, "text", COLUMN_DESCRIPTION);
+        column.set_sort_column_id(COLUMN_DESCRIPTION);
+        treeview.append_column(&column);
+    }
+
+    // Column for spinner
+    {
+        let renderer = CellRendererSpinner::new();
+        let column = TreeViewColumn::new();
+        column.pack_start(&renderer, true);
+        column.set_title("Spinning");
+        column.add_attribute(&renderer, "pulse", COLUMN_PULSE);
+        column.add_attribute(&renderer, "active", COLUMN_ACTIVE);
+        treeview.append_column(&column);
+    }
+
+    // Column for symbolic icon
+    {
+        let renderer = CellRendererPixbuf::new();
+        let column = TreeViewColumn::new();
+        column.pack_start(&renderer, true);
+        column.set_title("Symbolic icon");
+        column.add_attribute(&renderer, "icon-name", COLUMN_ICON);
+        column.add_attribute(&renderer, "sensitive", COLUMN_SENSITIVE);
+        column.set_sort_column_id(COLUMN_ICON);
+        treeview.append_column(&column);
+    }
+}
+
+fn spinner_timeout(model: &gtk::ListStore) -> Continue {
+    let iter = model.get_iter_first().unwrap();
+    let mut pulse = model.get_value(&iter, COLUMN_PULSE).get::<u32>().unwrap();
+    if pulse == u32::max_value() {
+        pulse = 0;
+    } else {
+        pulse += 1;
+    }
+
+    model.set_value(&iter, COLUMN_PULSE as u32, &pulse.to_value());
+    model.set_value(&iter, COLUMN_ACTIVE as u32, &true.to_value());
+
```

Comment:
You could do `model.set(&iter, &[COLUMN_PULSE, COUMN_ACTIVE], &[&pulse, &true]);` here